### PR TITLE
Linking and styling for evaluation

### PR DESCRIPTION
**Pull request by CarstenJahnke**
Watch Out my bugfix on my awesome MCU App 🤪

[🚀 get a little insight 🚀](https://mcu-akv5wwgt9-carstenjahnke.vercel.app/movies/1726) (Just for example)

### CODE REVIEW

File: components/styling/MovieDetailsStyling.js [hunk 1]
```diff
@@ -82,7 +93,23 @@ export const HeadingStyledContainer = styled.div`
...
 `;
+
+export const StyledSquareIcon = styled.div`
+  margin-top: 16px;
+  font-size: 10px;
+  float: right;
+`;
+
+export const StyledSquareIconImageWidth = 12;
+export const StyledSquareIconImageHeight = 12;
```
Comment (djfarly) [inline components/styling/MovieDetailsStyling.js:115]: It's a bit strange to user upper case variable names for non-components. There are some more in this file that were here before this PR. Please fix the naming for all widths and heights.

```suggestion
export const styledSquareIconImageWidth = 12;
export const styledSquareIconImageHeight = 12;
```

File: components/styling/MovieDetailsStyling.js [hunk 2]
```diff
@@ -82,7 +93,23 @@ export const HeadingStyledContainer = styled.div`
...
+export const StyledMovieCardReview = styled.div`
   ${StyledCard}
   ${StyledText}
+  padding: 6%;
+`;
+
+export const StyledMovieReview = styled.div`
+  ${StyledText}
+  text-align: center;
+  margin-bottom: 0px;
```
Comment (djfarly) [inline components/styling/MovieDetailsStyling.js:105]: 0 needs no unit

```suggestion
  margin-bottom: 0;
```

**Review by djfarly — APPROVED** [review]
Hej Carsten,

some small notes but nothing major ✅ 

Keep going. 🥳

**Comment by CarstenJahnke** [comment]
LGTM 🚀

**Pull request merged**